sw/airborne/modules/computer_vision/lib/vision/act_fast.c

```diff
@@ -22,16 +22,17 @@ All rights reserved.
  * de Croon, G.C.H.E. "ACT-FAST: efficiently finding corners by actively exploring images.", in submission.
  *
  */
-
 #include "fast_rosten.h"
 #include "act_fast.h"
 #include "math.h"
 #include "image.h"
 #include "../../opticflow/opticflow_calculator.h"
 
+// ACT-FAST agents arrays
 // equal to the maximal number of corners defined by fast9_rsize in opticflow_calculator.c
+// TODO Currently hardcoded to two cameras
 #define MAX_AGENTS FAST9_MAX_CORNERS
-struct agent_t agents[MAX_AGENTS];
+struct agent_t agents[2][MAX_AGENTS];
```

Comment:
Maybe make the number of cameras configurable ?

Reply:
I changed it so it checks if the second camera has been defined